quack_app/lib/core/menu_data.dart

```diff
@@ -24,46 +29,25 @@ class MenuData {
 
   // loadTodaysMenu loads the menu for the entire day
   Future loadTodaysMenu() async {
-    var foods = [
-      "Pancakes",
-      "Bagel",
-      "Cereal",
-      "Sandwhich",
-      "Wings",
-      "Grilled Chicken",
-      "Beyond Beef Burger",
-      "Portobello Mushrooms",
-      "Macaroni & Cheese",
-      "Veggie Fried Rice",
-      "Avocado Grilled Cheese",
-      "Mediterranean Pita",
-      "Pasta",
-      "French Fries",
-      "Ceasar Salad"
-    ];
-    _serveTimes = [
-      "Breakfast",
-      "Lunch",
-      "Dinner"
-    ]; // Serve times will be parsed here from backend
-    _stations = [
-      "Stem To Root",
-      "605 Kitchen",
-      "Spice",
-      "Luncheonnette"
-    ]; // Stations will be parsed here from backend
```

Comment:
Why was this comment removed?

Reply:
This comment is a reminder of what this PR implements